architecture/cicd_gitops.adoc

```diff
@@ -0,0 +1,62 @@
+
+You can use GitOps tooling to create repeatable and predictable processes for managing and recreating {product-title} clusters and applications. By using GitOps, you can address the issues of infrastructure and application configurations sprawl. It simplifies the propagation of infrastructure and application configurations changes across multiple clusters by defining your infrastructure and applications definitions as “code”. Implementing GitOps for your cluster configuration files can make automated installation easier and allow you to configure automated cluster customizations. You can apply the core principles of developing and maintaining software in a Git repo to the creation and management of your cluster and application configuration files.
+
+Automating both your cluster configuration and container development process by using GitOps allows you to pick and choose where and when to adopt this practice, using {product-title}. Using a CI pipeline that pairs with your GitOps strategy and execution plan is ideal. {product-title} provides the flexibility to choose when and how you integrate this methodology into your business practices and pipelines.
+
+GitOps works well with {product-title} because you can declaratively configure it and the state of the cluster configuration can be stored using Git. For more information, see xref:../installing/install_config/customizations.adoc#customizations[Available cluster customizations].
+
+[id="cicd_gitops_cluster_administration"]
+=== GitOps for single-cluster and multi-cluster administration
+
+Whether you need one or multiple independent or co-operative {product-title} clusters, you can use a GitOps strategy to manage the following tasks:
+
+* Ensure that these clusters have similar states for configuration, monitoring, or storage.
+* Recover or recreate clusters from a known state.
+* Create clusters with a known state.
+* Apply configuration changes to multiple {product-title} clusters.
+* Revert configuration changes to multiple {product-title} clusters.
+* Associate templated configuration with different environments.
+
+[id="cicd_gitops_application_configuration"]
+=== GitOps for application configuration management
+
+You can also use GitOps practices to manage application configuration. This practice ensures consistency in applications when you must deploy them to different clusters in different environments, like development, stage, and production, during their life cycle. Managing application configuration with GitOps is also beneficial when you must deploy applications across multiple clusters, whether on-cloud or on-premise, for availability and scalability purposes.
```

Comment:
What's the latest ruling on our usage of on-premise? I think this is it? I haven't seen "on-cloud" before. Maybe this sentence could be shortened to something like:
GitOps is also beneficial when deploying multiple cloud or on-premise clusters for greater availability and scalability.